docs/Measuring_Resonances.md

```diff
@@ -60,6 +58,32 @@ Fritzing wiring diagrams for some of the ADXL345 boards:
 
 ![ADXL345-Rpi](img/adxl345-fritzing.png)
 
+###### Via Raspberry Pi Pico
+
+You may connect the ADXL345 to your Raspberry Pi Pico and then connect the
+Pico to your Raspberry Pi via USB. This makes it easy to reuse the
+accelerometer on other Klipper devices, as you can connect via USB instead
+of GPIO.
+
+Please be particularly careful with your ADXL345. You want one without a
+level shifter and without being forced into I2C mode. For more details,
+see [this discourse post](https://klipper.discourse.group/t/raspberry-pi-pico-adxl345-portable-resonance-measurement/1757).
```

Comment:
I think it should be reworded, as it is OK to use an ADXL with level shifter as long as you do not connect it to VCC 5V.

Maybe something like:

> In order to avoid damage to your RPi make sure to connect the ADXL345 to 3.3V only. Depending on the board's layout, a level shifter may be present, which makes 5V dangerous for your RPi.